tap_anthem/streams.py

```diff
@@ -1,62 +1,149 @@
 """Stream type classes for tap-anthem."""
 
-from pathlib import Path
-from typing import Any, Dict, Optional, Union, List, Iterable
+from typing import Iterable
 
+import requests
 from singer_sdk import typing as th  # JSON Schema typing helpers
 
-from tap_anthem.client import anthemStream
+from tap_anthem.client import AnthemStream, FormularyNavigatorStream
 
-# TODO: Delete this is if not using json files for schema definition
-SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
-# TODO: - Override `UsersStream` and `GroupsStream` with your own stream definition.
-#       - Copy-paste as many times as needed to create multiple stream types.
 
-
-class UsersStream(anthemStream):
+class NhProvidersStream(AnthemStream):
```

Comment:
The [Anthem data](https://www22.anthem.com/cms-data-index.json/index.html) is broken up by state. Because I only care about New Hampshire data right now, I've built NH-specific streams. 

In the future, it may be useful to refactor these streams to make them more generic (i.e., handling any `Provider`, not just NH), thus making this tap usable for states outside of NH. This is assuming the shape of the data is consistent across states, which I haven't validated at this time.